slash/core/fixtures/utils.py

```diff
@@ -26,6 +26,15 @@ def fixture(func=None, name=None, scope=None, autouse=False):
         return yield_fixture(func=func, name=name, scope=scope, autouse=autouse)
     return _ensure_fixture_info(func=func, name=name, scope=scope, autouse=autouse)
 
+def use_fixtures(fixture_names):
+
+    def decorator(func):
+        func.__extrafixtures__ = fixture_names
```

Comment:
This doesn't support chaining, which might be a real use case in the future. For example:

```
def needs_something_initialized_globally(func):
    return use_fixtures('blap')(func)

@needs_something_initialized_globally
@use_fixtures('another')
def test_something():
    ....
```